spec/views/members/edit.html.erb_spec.rb

```diff
@@ -8,5 +8,14 @@
   it_behaves_like "a view without missing translations" do
     before { assign(:member, build_stubbed(:member)) }
   end
+
+  context 'it display the end_subscription' do
```

Comment:
delete this `context`, as it doesn't carry any more information than the `it`

Reply:
done